game/unicode.cc

```diff
@@ -12,7 +12,7 @@ void convertToUTF8(std::stringstream &_stream, std::string _filename) {
 		std::string data = _stream.str();
 		Glib::convert(data, "UTF-8", "UTF-8"); // Test if input is UTF-8
 		if (data.substr(0, 3) == "\xEF\xBB\xBF") {
-			std::clog << "unicode/warning: " << _filename << " UTF-8 BOM ignored. Please avoid editors that use BOMs (e.g. Notepad)." << std::endl;
+// 			std::clog << "unicode/warning: " << _filename << " UTF-8 BOM ignored. Please avoid editors that use BOMs (e.g. Notepad)." << std::endl;
```

Comment:
I'd love to get this uncommented again.
It's for us as dev's a very useful and the enduser won't notice it anyway.
Besides i'm using this to get my library txt files all UTF8 (without bom)

Reply:
Precisely as a dev I find it annoying as hell because it just clogs the log. I see a number of solutions. 1) creatr a boolean hidden configuration entry to enable them. 2) create a logging level more verbose than debug and demote these from warning to that level.

Reply:
Hmm i'd go for option 1 then to stay consitent with log levels. It may be a public bool tho. If the user wants to clean his file he can turn on song loading errors